### DISCUSSION TITLE
How to fix mc colors?

### DISCUSSION THREAD

**Discussion by xen** [ghostty-org/ghostty#3470] [Q&A] [answered]
This is comparison of Ghostty and old terminal I use:

<img width="1352" alt="Image" src="https://github.com/user-attachments/assets/036201a7-9506-4e63-b523-660f60d8cd4d" />

All other old terminals show nice dark blue colors for midnight commander. In Ghostty it is too light. How can I change it to the old style?

**Comment by jaredh159** [comment]
had a similar reaction, all the colors seemed "washed out" on my mac, compared to kitty. thought it was maybe just the background color so i set that to 000000 but still seems off.

**Reply by edrozenberg** [reply]
I noticed the same thing, I wound up using these colors for now, not quite perfect yet but better:
```
# Colors
palette = 04=#5763e1
palette = 06=#49a3b0
palette = 12=#0435f0
palette = 14=#69e2e4
```

Also noticed whatever colors I set in the Ghostty settings, the actual colors Ghostty displays are off some amount from the settings when I use a color picker to sample their hex codes.

**Reply by novadev94** [reply]
Try running `mc` with `TERM=xterm-256color`. I had a similar issue with `k9s`, and this fix works for me.

For more information, see: https://github.com/ghostty-org/ghostty/issues/336 and https://mitchellh.com/writing/ghostty-devlog-004 (the "We Found a Vim Bug! And Other Naughty Programs" section).

**Reply by net** [reply]
Try adding [`window-colorspace = display-p3`](https://ghostty.org/docs/config/reference#window-colorspace) to your config (macOS only).

**Comment by imran-iq** [comment]
Sort of having a similar issue with neovim

![2024-12-31_16-15](https://github.com/user-attachments/assets/9ccb14f7-bedf-42ab-87e3-9e5d30ebdd26)

Left (kitty), Right (ghostty) both on arch linux, whatever version is lastest in the repos as of writing this

It's the same code/configuration, but the left colours are all a little brighter (package/func/main) where as the colours on the right just look washed out

**Comment by xen** [comment, marked as answer]
I ended up with this configuration:

```
theme = deep
font-thicken = true
window-colorspace = display-p3
keybind = global:ctrl+grave_accent=toggle_quick_terminal
quick-terminal-animation-duration = 0.0
```

And this resolved my problem.

**Comment by mazksr** [comment]
I still have (the same?) the issue, even after applying `window-colorspace = display-p3`

<img width="1512" alt="Screenshot 2025-06-29 at 23 04 22" src="https://github.com/user-attachments/assets/a8f9f06f-fe18-4e78-9dc5-05ffc5d46f4d" />
Comparison with iterm on the left

Full config:
```
background = #191724
background-opacity = 0.85
background-blur = 54

foreground = #feffff

font-family = JetBrains Mono
font-style = SemiBold
font-size = 13

window-colorspace = display-p3
```

**Comment by sparkuss** [comment]
For better mc colors in ghostty for me it's working good this config:
```
# Tango palette
palette = 00=#2E3436
palette = 01=#CC0000
palette = 02=#4E9A06
palette = 03=#C4A000
palette = 04=#3465A4
palette = 05=#75507B
palette = 06=#06989A
palette = 07=#D3D7CF
palette = 08=#555753
palette = 09=#EF2929
palette = 10=#8AE234
palette = 11=#FCE94F
palette = 12=#729FCF
palette = 13=#AD7FA8
palette = 14=#34E2E2
palette = 15=#EEEEEC
```
